src/app/containers/FrontPageMain/index.jsx

```diff
@@ -44,22 +44,51 @@ const FrontPageMain = ({ frontPageData }) => {
         openGraphType="website"
       />
       <LinkedData type="WebPage" seoTitle={seoTitle} />
-      <main role="main">
-        <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
-          {offScreenText}
-        </VisuallyHiddenText>
-        <Grid>
-          <GridItemConstrainedLargeWithTopMargin>
-            {groups.map((group, index) => (
-              <FrontPageSection
-                key={group.title}
-                group={group}
-                sectionNumber={index}
-              />
-            ))}
-          </GridItemConstrainedLargeWithTopMargin>
+      <GridMaxWidthGroup4
+        as="main"
+        role="main"
+        columns={{
+          group0: 6,
+          group1: 6,
+          group2: 6,
+          group3: 6,
+          group4: 8,
+          group5: 20,
+        }}
+        enableGelGutters
+        enableGelMargins
+      >
+        <Grid
```

Comment:
I think this `Grid item` will have to be inside the `FrontPageSection`. Depending on the number of items in a group, we will we using different promos (top, regular, leading and without image) and wrapping them within `Grid items` with a different numbers of columns.